chat/slack.go

```diff
@@ -110,22 +107,22 @@ func (s *Slack) handleMessage(msg *slack.MessageEvent) {
 				MessageTS:  msg.Msg.Timestamp,
 			})
 			if err != nil {
-				fmt.Println(err)
+				s.logger.Println(err)
```

Comment:
We need to use different loglevels. For errors we should use Errorf or Error, for information messages we should use Infof or Info. Please change it